spec/spec.md

```diff
@@ -993,6 +993,23 @@ An error occurs (and no changes are made) if:
 * The coverage set is `campaign` and the scenario is `routine`. (All other combinations are acceptable)
 
 # Burden estimates
+## GET /modelling-groups/{modelling-group-id}/responsibilities/{touchstone-id}/{scenario-id}/model_run_parameter_sets/
```

Comment:
Model run parameter sets are not scenario specific. So this URL needs to change. This may require us to change other URLs, unfortunately. This is why I tried to spec things out in advance, but I didn't manage to forsee this!